zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/AuthorizationHandler.java

```diff
@@ -56,6 +56,7 @@ public void updateEntity(
       final Record<AuthorizationRecordValue> record, final AuthorizationEntity entity) {
     final AuthorizationRecordValue value = record.getValue();
     entity
+        .setId(String.valueOf(record.getKey()))
```

Comment:
❌ the id should have been set on `createNewEntity(id)` call. We should not change it here AFAICT. let's consult identity team as well.,